### DISCUSSION TITLE
Wierd Behavior when launched by ActiveSetup

### DISCUSSION THREAD

**Discussion by chrpai** [dotnet/wpf#9123] [Q&A]
I create a .net 8 windows exe that launches a full screen window with a background color of f2f2f2.  When I run it in Visual Studio, the compiled exe,  the compiled exe launched by the RunOnce command it works as expected.  However when the program is launched by ActiveSetup the background color is wrong. It's white.  I get same results if I use yellow as the background color.

I also tried putting a rectangle on the grid with a Fill color but the behavior was the same.  

Has anyone ever heard of something like this?  All the other controls on the window look fine.  It's just the background that gets rendered incorrectly. I can publish a repros that reproduces the issue if needed.

```
<Window x:Class="Someapp.MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        mc:Ignorable="d"
        Title="Some App" Height="1080" Width="1920" Icon="Icon.ico"  WindowStyle="None" WindowStartupLocation="CenterScreen" WindowState="Maximized" Background="yellow" Loaded="Window_Loaded" Topmost="true">
    <Grid>
        <Grid>
        </Grid>
    </Grid>
</Window>
```

PS- Tested on a Win10 and Win11 VM and it worked fine. It's just my own machine that it behaves this way. I tried turning off HDR and Dyanamic Lighting but it didn't help.  Very strange.

**Comment by chrpai** [comment]
I'm having this problem too!  Oh wait, it's just me finding my own issue again.  It still perplexes me.

**Reply by miloush** [reply]
What is ActiveSetup?

**Reply by Symbai** [reply]
Active Setup is a mechanism for executing commands once per user early during [login](https://en.wikipedia.org/wiki/Login). Active Setup is used by some [Microsoft Windows](https://en.wikipedia.org/wiki/Microsoft_Windows) [operating system](https://en.wikipedia.org/wiki/Operating_system) components like [Internet Explorer](https://en.wikipedia.org/wiki/Internet_Explorer) to set up an initial configuration for new users logging on for the first time. Active Setup is also used in some corporations’ software distribution systems to create an initial customized user environment

Source: https://en.wikipedia.org/wiki/Active_Setup
